oap-server/server-starter/src/main/resources/bydb.yml

```
@@ -52,15 +52,46 @@ groups:
   # The "normal" section defines settings for datasets not specified in "super".
   # Each dataset will be grouped under a single group named "normal".
   recordsNormal:
+    # The settings for the default "hot" stage.
     shardNum:  ${SW_STORAGE_BANYANDB_GR_NORMAL_SHARD_NUM:1}
     segmentInterval: ${SW_STORAGE_BANYANDB_GR_NORMAL_SI_DAYS:1}
     ttl: ${SW_STORAGE_BANYANDB_GR_NORMAL_TTL_DAYS:3}
+    # If the "warm" stage is enabled, the data will be moved to the "warm" stage after the TTL of the "hot" stage.
+    # If the "cold" stage is enabled and "warm" stage is disabled, the data will be moved to the "cold" stage after the TTL of the "hot" stage.
+    # If both "warm" and "cold" stages are enabled, the data will be moved to the "warm" stage after the TTL of the "hot" stage, and then to the "cold" stage after the TTL of the "warm" stage.
+    # OAP will query the data from the "hot and warm" stage by default if the "warm" stage is enabled.
+    enableWarmStage: ${SW_STORAGE_BANYANDB_GR_NORMAL_ENABLE_WARM_STAGE:true}
+    enableColdStage: ${SW_STORAGE_BANYANDB_GR_NORMAL_ENABLE_COLD_STAGE:true}
+    # The settings for the "warm" stage.
+    warm:
+      shardNum: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_SHARD_NUM:1}
+      segmentInterval: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_SI_DAYS:1}
```

normal record should not have huge scale of data, let' keep the file less.
```suggestion
      segmentInterval: ${SW_STORAGE_BANYANDB_GR_NORMAL_WARM_SI_DAYS:2}
```